language/tools/transaction-replay/src/lib.rs

```diff
@@ -165,12 +166,14 @@ impl LibraDebugger {
         let is_version_ok = |version| {
             self.run_session_at_version(version, |session| {
                 let mut cost_strategy = CostStrategy::system(&gas_table, GasUnits::new(0));
+                let logger = StdErrLogger;
```

Comment:
here also we are going to stderror, is that ok? another way to do all of those is to turn on logging on a normal log.
Thoughts? @runtian-zhou

Reply:
Sounds good to me!